src/main/java/org/fenixedu/learning/domain/executionCourse/ExecutionCourseSite.java

```diff
@@ -56,13 +58,16 @@
     public ExecutionCourseSite(ExecutionCourse executionCourse) {
         checkNotNull(executionCourse);
         setExecutionCourse(executionCourse);
-        setPublished(true);
+
         setFolder(folderForPath(PortalConfiguration.getInstance().getMenu(), "courses"));
         setSlug(on("-").join(getExecutionCourse().getSigla(), getExecutionCourse().getExternalId()));
-        setCanAdminGroup(NobodyGroup.get());
-        setCanPostGroup(NobodyGroup.get());
-        setBennu(Bennu.getInstance());
 
+        setCreationDate(new DateTime());
```

Comment:
This comment leaves everything to be guessed, again.
I do not know why it shouldn't be set here. The creation time-stamp being set inside the constructor of a certain entity is a simple, natural and widely accepted practice. So natural it even feels silly having to justify something like this.
Nevertheless, if a reasonable argument and an alternative are provided we will gladly indulge you.